backend/routes/event_routes/user_event_routes.py

```
@@ -0,0 +1,139 @@
+        limit=limit,
+        cursor_scheduled_date=cursor_scheduled_date,
+        cursor_id=cursor_id,
+    )
+
+@private_event_router.get("/private-event-instance-details/{instance_id}")
+async def get_private_instance_details(request: Request, instance_id:str):
+    return await get_instance_details(instance_id=instance_id, uid=request.state.uid, favorite_events=request.state.user.get("favorite_events"))
+
+
+@public_event_router.get("/public-event-instance-details/{instance_id}")
+async def get_public_instance_details(request: Request, instance_id:str):
```

Ruff (0.14.4)

Unused function argument: `request`

(ARG001)